test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -53,19 +63,25 @@ export default class PossibleIssues extends Component {
         if (error) {
             return <div>Error: {error}</div>;
         } else {
-            const columns = [{
-                title: 'Possible Issues',
-                dataIndex: 'issues',
-                key: 'issues',
-            }];
+            let generateColumns = (inputRepoName) => {
+                return [{
+                    title: inputRepoName,
+                    dataIndex: 'issues',
+                    key: 'issues',
+                }]
+            };
             return <div>
                 <TestBreadcrumb buildId={buildId} testId={testId} testName={testName} />
-                <Table
-                    columns={columns}
-                    dataSource={dataSource}
-                    bordered
-                    title={() => testName}
-                />
+                {
+                    Object.keys(dataSource).map((repoName, index) => (
+                        <Table 
+                            key={index}
+                            columns={generateColumns(repoName)}
+                            dataSource={dataSource[repoName]}
+                            bordered
```

Comment:
Original code can still be used. We do not need to change line 66 - line 72. Just need to update `title`:

```
<Table
   columns={columns}
   dataSource={dataSource[repoName]}
   bordered
   title={() => repoName}
...
```